finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,21 @@
+for(var i = 10; i > 0; i--){
```

Comment:
You're missing one case here, I believe. When I ran your code, one part of the rhyme printed the following

> 2 speckled frogs sat on a log eating some most delicious bugs. One jumped in the pool where its nice and cool, then **there were 1 speckled frogs**.

How could you update this code to correct the above statement?

Reply:
@mschae16 I totally missed that sorry! I updated my code with the lines below and that should fix the issue!

if(i - 1 > 1){
    console.log(i + " speckled frogs sat on a log eating some most delicious bugs. One jumped in the pool where its nice and cool, then there were " + (i - 1) +" speckled frogs.")
  } else if (i === 2){
    console.log(i + " speckled frogs sat on a log eating some most delicious bugs. One jumped in the pool where its nice and cool, then there was " + (i - 1) +" speckled frogs.")